feconf.py

```diff
@@ -40,6 +40,7 @@
 else:
     raise Exception('Invalid platform: expected one of [\'gae\']')
 
+CACHE_SLUG = 'dev'
```

Comment:
CACHE_SLUG_DEV, and add a comment above it explaining what it's for. It is OK for the comment to be verbose; prioritize clarity here.